packages/css-data/src/__generated__/animatable-properties.ts

```diff
@@ -51,16 +51,10 @@ export const animatableProperties = [
   "column-rule-color",
   "column-rule-width",
   "column-width",
-  "contain-intrinsic-block-size",
```

Comment:
pnpm install and run pnpm build:mdn-data again pls

Reply:
done